x-pack/plugin/vector-tile/src/main/java/org/elasticsearch/xpack/vectortile/rest/RestVectorTileAction.java

```diff
@@ -145,6 +145,8 @@ public RestResponse buildResponse(SearchResponse searchResponse) throws Exceptio
                                 .filter(a -> GRID_FIELD.equals(a.getName()) == false && BOUNDS_FIELD.equals(a.getName()) == false)
                                 .collect(Collectors.toList())
                         );
+                    // TODO: I think this is copying a Search Response to strip out the hits,
+                    // but took should have been captured at the transport layer so no need to measure here.
```

Comment:
@javanna I'm hoping to confirm my understanding that we don't need instrumentation here, as "took"times should have already been recorded by the original response prior to this point, correct?